packages/react-devtools-inline/__tests__/End-to-End/todoList.test.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
```

Comment:
File structure nit: Can we put these tests in a folder with the naming pattern `"__tests__/__e2e__"`?

Also "todoList.test" isn't a very meaningful name. What is this test _testing_? Seems like it's testing that props are correct when inspecting components. So maybe we could name it something more like `"inspecting-props.js"`? Something that more clearly describes its purpose

Reply:
Actually I thought of organinising all the apps tests in one test file. Like in `devtools shell` there are many apps like so `todoList.test.js` will have all tests related to todo list app.

Well I have no problem in changing to this name pattern.

Also we can surely change the name to `__e2e__`

Reply:
My gut is that organizing tests around _what they're testing_ is more useful when you're looking to add new ones (or verify that a change didn't break an old one)

Reply:
Sure that is comfortable to navigate into. Also if any apps contains dozen of tests it's difficult to lookup for a specific test for improvement later on.